src/ShadowDOM/wrappers/Selection.js

```diff
@@ -56,6 +53,15 @@
     }
   };
 
+  // Not all browsers support Selection.extend. Some versions of IE do not support extend, code that checks
+  // if extend exists in the Selection would fail the test if we define extend on the wrapper and it does not exist in
```

Comment:
fyi, the JS style guide this code used (https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml) normally has comments wrapping at 80 columns.

(apologies for nitpicking style. I don't personally feel it's a big deal, but a lot of folks do, so as a reviewer I try and remember to check for it.)